src/CVRF/cvrf_enumeration.c

```diff
@@ -0,0 +1,178 @@
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <string.h>
+
+#include <libxml/xmlreader.h>
+#include <libxml/xmlwriter.h>
+
+#include "public/cvrf.h"
+#include "cvrf_priv.h"
+
+#include "common/list.h"
+#include "common/_error.h"
+#include "common/xmltext_priv.h"
+#include "common/elements.h"
+#include "common/oscap_string.h"
+#include "common/util.h"
+
+#include "source/oscap_source_priv.h"
+#include "source/public/oscap_source.h"
+
+
+/*****************************************************************************
+ * Static functions
+ */
+
+static int cvrf_enumeration_attr(xmlTextReaderPtr reader, char *attname, const struct oscap_string_map *map) {
```

Comment:
ping, this has been ignored and needs to be fixed. we need type safety for enums.